static/src/stylesheets/module/content/_content.scss

```diff
@@ -154,6 +154,22 @@
     }
 }
 
+// Styling for long section/series labels at the leftCol breakpoint
+.content__labels--long {
+    @include mq(leftCol, wide) {
+        .content__section-label__link {
+            @include fs-header(2);
+            line-height: 22px;
```

Comment:
Or rather than px use relative units: `line-height: 1.2` The calculations are more consistent across the browsers with relative line-height and it won't add line-height with rem to the final CSS.